src/odemis/acq/test/move_test.py

```diff
@@ -490,6 +556,44 @@ def test_unknown_label_at_initialization(self):
         self.assertEqual(current_grid, None)
 
 
+class TestMeteorZeiss1Move(TestMeteorMove):
+    """
+    """
+    MIC_CONFIG = METEOR_ZEISS1_CONFIG
+    ROTATION_AXES = {'rx', 'rm'}
+
+    def test_moving_in_grid1_fm_imaging_area_after_loading(self):
```

Comment:
If all the beginning is the same as the whole function of the parent class, you call it, instead of duplicating the code:
`super().test_...()`